File: exercises-cpp/3. Repetitivas/20.cpp
#include<iostream>
using namespace std;

int main(){
    system("cls");
    int lista[10];

    for(int i=0; i<10; i++ ){
        cout<<"Numero "<<i+1<<": "; cin>>lista[i];
    }

    int aux, min;
    for(int i=0; i<10; i++){
        min = i;
        for(int j=i+1; j<10; j++){
            if( lista[j] < lista[min] ) { min = j; }
        }
        aux = lista[i];
        lista[i] = lista[min];
        lista[min] = aux;
    }

    cout<<"Promedio mayor: "<<(lista[8]+lista[9])/2<<endl;
    cout<<"Promedio menor: "<<(lista[0]+lista[1])/2<<endl;

    return 0;
}

File: exercises-python/3. Repetitivos/20.py
'''
20.Diseñe el programa que muestre el Menor, Mayor, Promedio de una lista de 10 números.
'''
import os
os.system("cls")

lista = []
mayor = 0
menor = 0
promedio = 0
suma = 0
print("Dame 10 números:",end="")
for i in range(10):
    lista.append(int(input()))
mayor = max(lista)
menor = min(lista)

for i in lista:
    suma = suma + i
    promedio = suma/10
print(f"Mayor: {mayor}")
print(f"Menor: {menor}")
print(f"Promedio: {promedio}")